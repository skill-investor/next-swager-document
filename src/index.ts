export { withSwagger } from './with-swagger';
