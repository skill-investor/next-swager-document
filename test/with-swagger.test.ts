import { prepareSwaggerOptions } from '../src/with-swagger';

describe('withSwagger', () => {
  it('should create default swagger json option', () => {
    expect(
      prepareSwaggerOptions({
        title: 'NextJS Swagger',
        version: '0.1.0',
      })
    ).toMatchSnapshot();
  });
});
