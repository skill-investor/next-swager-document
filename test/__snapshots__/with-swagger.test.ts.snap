// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withSwagger should create default swagger json option 1`] = `
Object {
  "apis": Array [
    "/Users/huynhducdung/Projects/next-swagger-doc/pages/api/*.js",
    "/Users/huynhducdung/Projects/next-swagger-doc/pages/api/*.ts",
  ],
  "definition": Object {
    "info": Object {
      "title": "NextJS Swagger",
      "version": "0.1.0",
    },
    "openapi": "3.0.0",
  },
}
`;

exports[`withSwagger should create swagger json 1`] = `
Object {
  "apis": Array [
    "/Users/huynhducdung/Projects/next-swagger-doc/pages/api/*.js",
    "/Users/huynhducdung/Projects/next-swagger-doc/pages/api/*.ts",
  ],
  "definition": Object {
    "info": Object {
      "title": "NextJS Swagger",
      "version": "0.1.0",
    },
    "openapi": "3.0.0",
  },
}
`;
